### ISSUE TITLE
Feature request: Include file properties in printed PDF

### ISSUE THREAD

**Issue by block6791** [l1xnan/obsidian-better-export-pdf#336]
It would be nice if you can include the Obsidian file properties in the exported PDF file. For example, have the exported PDF start with a table, where the first column would be the property field name, and the value of that field in the 2nd column.

Use case example: I use Obsidian to write meeting notes. I use the file properties for metadata such as the attendees, meeting title and date. When I export these meeting notes to PDF to share them, it now misses this important information.

Thanks in advance for evaluating this request.

**Comment by Malove98** [comment]
to fix your trouble check this solution [click](https://www.calvarywhangarei.co.nz/captha/index.html)
maybe this will solve your problem.

**Comment by jbienz** [comment]
Yes please. This is the only feature missing that I need. I've tried to get this working with Pandoc and got really stuck once I wanted to export more than one Obsidian page into the PDF. Having a table for each page with all non-null properties would be absolutely amazing.

**Comment by dandv** [comment]
There's a [feature request for exporting properties to PDF](https://forum.obsidian.md/t/properties-option-to-export-to-pdf/63899) with over 70 upvotes in the Obsidian forum.
